app/screens/edit_profile/edit_profile.js

```diff
@@ -488,6 +490,9 @@ export default class EditProfile extends PureComponent {
                         <View style={style.top}>
                             <AttachmentButton
                                 blurTextBox={emptyFunction}
+                                browseFileTypes={DocumentPickerUtil.images()}
+                                canTakeVideo={false}
+                                maxFileSize={MAX_SIZE}
```

Comment:
Why the 2MB restriction here when web max file size is about 190MB

Reply:
@sudheerDev for the profile image? I couldn't find it anywhere, can you point me to that setting?

Reply:
@enahum i just checked in the UI but 2mb seems low isn't it?

<img width="567" alt="screenshot 2018-12-10 at 5 33 34 pm" src="https://user-images.githubusercontent.com/4973621/49731570-d0ee1c00-fca1-11e8-9439-4cfe490ac1f5.png">

Reply:
Yes 2MB is low but 190 seems excessive for a mobile app, anyway I'll see if there is a config for this

Reply:
@sudheerDev it seems that it is using the config `MaxFileSize` that is used for regular uploads, I think that can be excessive for a profile picture, what if we set it to 20Mb?

Reply:
Yup, 20mb sounds good to me. Thanks

Reply:
@sudheerDev done